conan/tools/qbs/qbs.py

```diff
@@ -19,7 +19,16 @@ def _configuration_dict_to_commandlist(name, config_dict):
 
 
 class Qbs(object):
+    '''
```

Comment:
PyCharm is complaining that docstrings should use double qoutes as per PEP 257, even though it's not really an issue, changing it would remove noise from the IDE :)